assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -61,16 +57,16 @@ export default function useMediaPicker({
   });
   useEffect(() => {
     try {
+      // Handles the video processing logic from 3rd party.
+      // The 'success' listener logic will be triggred once user uploaded a
+      // file from the uploader.
```

Comment:
"The Uploader.success callback is invoked when a user uploads a file."

Also please add details about the race condition for future reference. I.e. under what circumstances is it not called?

Reply:
Updated the comment.